pastefox-share/src/PasteFoxSharePlugin.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace FlexKleks\PasteFoxShare;
+
+use App\Contracts\Plugins\HasPluginSettings;
+use App\Traits\EnvironmentWriterTrait;
+use Filament\Contracts\Plugin;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Components\TextInput;
+use Filament\Notifications\Notification;
+use Filament\Panel;
+use Filament\Schemas\Components\Section;
+
+class PasteFoxSharePlugin implements HasPluginSettings, Plugin
+{
+    use EnvironmentWriterTrait;
+
+    public function getId(): string
+    {
+        return 'pastefox-share';
+    }
+
+    public function register(Panel $panel): void
+    {
+        if ($panel->getId() === 'server') {
+            $id = str($panel->getId())->title();
+
+            $panel->discoverPages(
+                plugin_path($this->getId(), "src/Filament/$id/Pages"),
+                "FlexKleks\\PasteFoxShare\\Filament\\$id\\Pages"
+            );
+        }
+    }
+
+    public function boot(Panel $panel): void
+    {
+        //
+    }
```

Comment:
```suggestion
    public function register(Panel $panel): void {}

    public function boot(Panel $panel): void {}
```
The plugin no longer has pages.